common/lib/capa/capa/tests/test_input_templates.py

```diff
@@ -314,6 +303,33 @@ def test_problem_marked_incorrect(self):
                                  "//label[@class='choicegroup_correct']",
                                  self.context)
 
+    def test_radio_problem_marked_incorrect(self):
+        """
+        Test all conditions under which the entire problem
+        (not a particular option) is marked incorrect.
+        """
+        conditions = [
+            {'status': Status('incorrect'), 'input_type': 'radio', 'value': ''},
+            {'status': Status('incomplete'), 'input_type': 'radio', 'value': ''}
+        ]
+
+        for test_conditions in conditions:
+            self.context.update(test_conditions)
+            xml = self.render_to_xml(self.context)
+            xpath = "//div[@class='indicator-container']/span[@class='status incorrect']"
+
+            # The status should now be on the choice after a submission
+            self.assert_no_xpath(xml, xpath, self.context)
```

Comment:
Having tests with only assert_no_xpath concerns me-- the xpath selector could simply be bogus, and there is nothing to catch that. Can you also submit an answer and then verify that the same xpath DOES exist?